saleor/graphql/webhook/tests/mutations/test_delivery_retry.py

```diff
@@ -0,0 +1,102 @@
+from unittest.mock import patch
+
+import graphene
+
+from saleor.graphql.tests.utils import assert_no_permission, get_graphql_content
+
+WEBHOOK_DELIVERY_RETRY_MUTATION = """
+    mutation eventDeliveryRetry($id: ID!){
+      eventDeliveryRetry(id: $id){
+        errors{
+          field
+          message
+        }
+        delivery{
+          id
+        }
+      }
+    }
+"""
+
+
+@patch("saleor.plugins.manager.PluginsManager.event_delivery_retry")
+def test_delivery_retry_mutation(
+    mocked_send_request_async,
+    app_api_client,
+    permission_manage_apps,
+    event_delivery,
+    settings,
+):
+    # given
+    settings.PLUGINS = ["saleor.plugins.webhook.plugin.WebhookPlugin"]
```

Comment:
Do we need to enable webhook plugin here? Looks like we mock a PluginsManager's method here (which is great) so, no need to include here a webhook Plugin.